templates/core/index.html

```diff
@@ -1,8 +1,10 @@
 {% extends "core/base_core.html" %}
 
 {% block title %}Ubuntu Core{% endblock %}
-{% block meta_description %}Ubuntu Core is Ubuntu for embedded environments, optimised for security and reliable updates. Run snaps in a high-security confined sandbox with bulletproof upgrades.{% endblock meta_description %}
-{% block meta_copydoc %}https://docs.google.com/document/d/1QoM0Yqx4JfDJ2G3OPt5bds-ximG1jASmU_PWX4iff3E/edit{% endblock meta_copydoc %}
+{% block meta_description %}Ubuntu Core is Ubuntu for embedded environments, optimised for security and reliable
```

Comment:
you have newlines here, is your editor 'hard-wrapping' we don't like this, well, I don't.